src/common/components/spec/centered.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import * as React from 'react';
+import { CenteredContainer } from '../centered';
+
+describe('CenteredContainer', () => {
+    const plain = str => str.split(/(?:\r\n|\r|\n)/g).map(line => line.trim()).join('');
```

Comment:
Now you don't need to add this harcoded regex, we have already @atobajas and @crsanti refactor in master, just use the functin that you can find under:

_./src/parse/multilinetrim.ts_

e.g. 

```
const plain = multilineTrim(expectedDomTree);
```

Well in this test is not used (remove it), in the rest of test where is used just replace it (this new method has it's own unit tests)